pyproject.toml

```diff
@@ -39,48 +33,51 @@ Documentation = "https://docs.temporal.io/docs/python"
 
 [dependency-groups]
 dev = [
-    "cibuildwheel>=2.22.0,<3",
-    "grpcio-tools>=1.48.2,<2",
-    "mypy==1.4.1",
-    "mypy-protobuf>=3.3.0,<4",
-    "psutil>=5.9.3,<6",
-    "pydocstyle>=6.3.0,<7",
-    "pydoctor>=24.11.1,<25",
-    "pyright==1.1.403",
-    "pytest~=7.4",
-    "pytest-asyncio>=0.21,<0.22",
-    "pytest-timeout~=2.2",
-    "ruff>=0.5.0,<0.6",
-    "toml>=0.10.2,<0.11",
-    "twine>=4.0.1,<5",
-    "ruff>=0.5.0,<0.6",
-    "maturin>=1.8.2",
-    "pytest-cov>=6.1.1",
-    "httpx>=0.28.1",
-    "pytest-pretty>=1.3.0",
-    "openai-agents[litellm]>=0.3,<0.4"
+  "cibuildwheel>=2.22.0,<3",
+  "grpcio-tools>=1.48.2,<2",
+  "mypy==1.4.1",
+  "mypy-protobuf>=3.3.0,<4",
+  "psutil>=5.9.3,<6",
+  "pydocstyle>=6.3.0,<7",
+  "pydoctor>=24.11.1,<25",
+  "pyright==1.1.403",
+  "pytest~=7.4",
+  "pytest-asyncio>=0.21,<0.22",
+  "pytest-timeout~=2.2",
+  "ruff>=0.5.0,<0.6",
+  "toml>=0.10.2,<0.11",
+  "twine>=4.0.1,<5",
+  "maturin>=1.8.2",
+  "pytest-cov>=6.1.1",
+  "httpx>=0.28.1",
+  "pytest-pretty>=1.3.0",
+  "openai-agents[litellm]>=0.3,<0.4",
+  "googleapis-common-protos==1.70.0",
 ]
 
 [tool.poe.tasks]
 build-develop = "uv run maturin develop --uv"
 build-develop-with-release = { cmd = "uv run maturin develop --release --uv" }
-format = [{cmd = "uv run ruff check --select I --fix"}, {cmd = "uv run ruff format"}, ]
+format = [
+  { cmd = "uv run ruff check --select I --fix" },
+  { cmd = "uv run ruff format" },
+]
 gen-docs = "uv run scripts/gen_docs.py"
 gen-protos = "uv run scripts/gen_protos.py"
 gen-protos-docker = "uv run scripts/gen_protos_docker.py"
```

Comment:
@VegetarianOrc - to confirm, we decided to move the simple uv-run type of steps out of gen_protos_docker any as poe sub-commands to these two commands right?